### ISSUE TITLE
Doesn't work in IE on tablet with Windows OS

### ISSUE THREAD

**Issue by majco333** [rochal/jQuery-slimScroll#154]
Hi,

your approach doesn't work in IE10 on tablet with Windows OS.
Is it possible to fix it?

Thanks

Majo

**Comment by philipstratford** [comment]
Agreed, the scrolling functionality is completely absent on my Surface, both in the desktop and Windows Store versions of IE.
